src/dataframerow/dataframerow.jl

```diff
@@ -57,7 +57,18 @@ function Base.iterate(r::DataFrameRow, st)
     return ((_names(r)[st], r[st]), st + 1)
 end
 
-Base.convert(::Type{Array}, r::DataFrameRow) = convert(Array, parent(r)[row(r),:])
+function Base.convert(::Type{Array}, r::DataFrameRow)
+    Base.depwarn("Conversion of `DataFrameRow` to a `Matrix` is deprecated. " *
+                 "`Vector` will be returned in the future.", :convert)
+    convert(Array, parent(r)[row(r),:])
+end
+Base.convert(::Type{Vector}, dfr::DataFrameRow) =
+    [dfr[i] for i in 1:length(dfr)]
+Base.convert(::Type{Vector{T}}, dfr::DataFrameRow) where T =
+    T[dfr[i] for i in 1:length(dfr)]
+Base.Vector(dfr::DataFrameRow) = Base.convert(Vector, df)
```

Comment:
No need for `Base.`.

Reply:
Interesting - I did not think earlier abut such case. Where is this documented?

Reply:
Well, you never need to qualify calls to exported functions from other modules. You don't write `Base.length` either, do you? :-)

Reply:
Also, the CI failure is due to `df` instead of `dfr`.

Reply:
Sure - I thought you wanted to strip `Base.` from `Base.Vector` and got confused.